web/client/components/widgets/builder/wizard/common/WPSWidgetOptions.jsx

```diff
@@ -311,7 +311,7 @@ export default ({
                                 onChange("autoColorOptions.classification", formatAutoColorOptions(newClassification, attributeType) || []);
                             }
                         }}
-                        options={typedOptions}
+                        options={typedOptions.filter(item => item.type === 'string' || item.type === 'number')}
```

Comment:
based on previous comment shouldn't be `options.filter(...` instead?

Reply:
correct, I missed this change. Fixed